### ISSUE TITLE
Angular dimension not moving during a copy, move, or drag.

### ISSUE THREAD

**Issue by labusker** [LibreCAD/LibreCAD#1019]
![examplebeforemove](https://user-images.githubusercontent.com/44000518/46819356-257c2600-cd49-11e8-899e-1ed5bd80a562.PNG)
![exampleaftermove](https://user-images.githubusercontent.com/44000518/46819365-29a84380-cd49-11e8-9650-e7e857de8155.PNG)

**Comment by labusker** [comment]
When an angular dimension is used in a drawing and then copied or moved to another position, the angular does not move with the other objects. If I save the file, close it, then reopen it, the angulars are then shown to be moved correctly. This will happen 100% of the time on my windows 10 computer using librecad 2.2.0-rc1. I installed 2.1.3 on my surface computer and angular dimensions move correctly. I then updated to 2.2.0-rc1 and then had the same angular move problem. I uninstalled 2.2.0, then reinstalled 2.1.3 and all worked fine.
